Documentation/docs/contributing/index.md

```diff
@@ -344,7 +346,7 @@ git merge --no-ff release-X.X
 git push upstream release
 ```
 
-then:
+4. Finally, merge the `release` branch back into the `master` branch:
```

Comment:
When a bug fix is cherry-picked from the `master` and then pushed onto the `release` branch, is it still useful to merge the `release` branch back to the `master`?

Reply:
Yes, conflicts should be addressed and the Git commit history, a directed acyclic graph (DAG) and Merkle Tree, should keep the master, release, release-X.X branches connected.